php/sync/class-storage.php

```diff
@@ -182,10 +182,14 @@ public function sync( $attachment_id ) {
 
 		// Get the previous state of the attachment.
 		$previous_state = $this->media->get_post_meta( $attachment_id, Sync::META_KEYS['storage'], true );
+
+		// don't apply the default transformations here.
+		add_filter( 'cloudinary_apply_default_transformations', '__return_false' );
+
 		switch ( $this->settings['offload'] ) {
 			case 'cld':
 				$this->remove_local_assets( $attachment_id );
-				update_post_meta( $attachment_id, '_wp_attached_file', $this->media->cloudinary_url( $attachment_id ) );
+				update_post_meta( $attachment_id, '_wp_attached_file', $this->media->cloudinary_url( $attachment_id, array(), array(), null, false ) );
```

Comment:
@dugajean 

Same here. Seems that we don't need all these arguments added. Can you please confirm?